mongomock/collection.py

```diff
@@ -1581,11 +1581,23 @@ def _handle_conditional_operator(operator, values, doc_dict):
                 except KeyError:
                     return fallback
                 return out_value if out_value is not None else fallback
-            else:
-                raise NotImplementedError(
-                    "Although '%s' is a valid date operator for the "
-                    "aggregation pipeline, it is currently not implemented "
-                    " in Mongomock." % operator)
+            if operator == '$cond':
```

Comment:
To be in line with how the other types of operators, and the stages of the aggregation pipeline, are handled, I'd suggest using `elif` here (and maybe keeping the `else` for `NotImplementedError`, even though it is the very last statement in the method).

Reply:
Done.